dev/Generated/AnimatedVisualPlayer.properties.cpp

```diff
@@ -0,0 +1,230 @@
+                nullptr);
+    }
+    if (!s_DurationProperty)
+    {
+        s_DurationProperty =
+            InitializeDependencyProperty(
+                L"Duration",
+                winrt::name_of<winrt::TimeSpan>(),
+                winrt::name_of<winrt::AnimatedVisualPlayer>(),
+                false /* isAttached */,
+                ValueHelper<winrt::TimeSpan>::BoxedDefaultValue(),
+                nullptr);
+    }
+    if (!s_FallbackContentProperty)
+    {
+        s_FallbackContentProperty =
+            InitializeDependencyProperty(
+                L"FallbackContent",
+                winrt::name_of<winrt::DataTemplate>(),
+                winrt::name_of<winrt::AnimatedVisualPlayer>(),
+                false /* isAttached */,
+                ValueHelper<winrt::DataTemplate>::BoxedDefaultValue(),
+                nullptr);
+    }
+    if (!s_IsAnimatedVisualLoadedProperty)
+    {
+        s_IsAnimatedVisualLoadedProperty =
+            InitializeDependencyProperty(
+                L"IsAnimatedVisualLoaded",
+                winrt::name_of<bool>(),
+                winrt::name_of<winrt::AnimatedVisualPlayer>(),
+                false /* isAttached */,
+                ValueHelper<bool>::BoxedDefaultValue(),
+                nullptr);
+    }
+    if (!s_IsPlayingProperty)
+    {
+        s_IsPlayingProperty =
+            InitializeDependencyProperty(
+                L"IsPlaying",
+                winrt::name_of<bool>(),
+                winrt::name_of<winrt::AnimatedVisualPlayer>(),
+                false /* isAttached */,
+                ValueHelper<bool>::BoxedDefaultValue(),
+                nullptr);
+    }
+    if (!s_PlaybackRateProperty)
+    {
+        s_PlaybackRateProperty =
+            InitializeDependencyProperty(
+                L"PlaybackRate",
+                winrt::name_of<double>(),
+                winrt::name_of<winrt::AnimatedVisualPlayer>(),
+                false /* isAttached */,
+                ValueHelper<double>::BoxedDefaultValue(),
+                nullptr);
+    }
+    if (!s_SourceProperty)
+    {
+        s_SourceProperty =
+            InitializeDependencyProperty(
+                L"Source",
+                winrt::name_of<winrt::IAnimatedVisualSource>(),
+                winrt::name_of<winrt::AnimatedVisualPlayer>(),
+                false /* isAttached */,
+                ValueHelper<winrt::IAnimatedVisualSource>::BoxedDefaultValue(),
+                nullptr);
+    }
+    if (!s_StretchProperty)
+    {
+        s_StretchProperty =
+            InitializeDependencyProperty(
+                L"Stretch",
+                winrt::name_of<winrt::Stretch>(),
+                winrt::name_of<winrt::AnimatedVisualPlayer>(),
+                false /* isAttached */,
+                ValueHelper<winrt::Stretch>::BoxedDefaultValue(),
+                nullptr);
+    }
+}
+
+void AnimatedVisualPlayerProperties::ClearProperties()
+{
+    s_AutoPlayProperty = nullptr;
```

Comment:
@simeoncran , is it intentional not to clear s_AutoPlayProperty in the old code?

Reply:
I'm sure it was an oversight, that's why we generate this code. It was too easy to get wrong.